src/main/groovy/com/epam/petclinic/plugin/QualityAwareJavaPlugin.groovy

```diff
@@ -133,22 +139,52 @@ public class QualityAwareJavaPlugin implements Plugin<Project> {
                     project.ant.xslt(
                             in: "$xmlReportPath",
                             out: "$htmlReportPath",
-                            style: getToolPath(project, stylesheetRelativePath),
+                            style: getFilePath(project, stylesheetRelativePath),
                             destdir: "${project[toolName].reportsDir}"
                     )
                 }
             }
         }
     }
 
-    private static TextResource getToolResource(Project project, String relativeReference) {
+    private static TextResource getToolResource(Project project, String relativeReference, String overriddenPath) {
         return project.rootProject.resources.text.fromFile(
-                getToolPath(project, relativeReference)
+                getFilePath(project, relativeReference, overriddenPath)
         )
     }
 
-    private static String getToolPath(Project project, String relativeReference) {
-        return "${project.rootDir}/${CODE_QUALITY_DIR}/${relativeReference}"
+    private static TextResource getToolResource(Project project, String relativeReference) {
+        return getToolResource(project, relativeReference, null)
+    }
+
+    private static String getFilePath(Project project, String localPath, String overriddenPath) {
+        if (overriddenPath == null || overriddenPath.isEmpty()) {
+            // Gets file from plugins resources locally and copy to project
+            return getAndCopyFileFromPluginsToProject(project, localPath)
```

Comment:
Yes, if we will have the main project, it should be one dir, but now it is required for idea plugins and anyway, they should be copied